src/javascript/EditPanel/EditPanelContent/FormBuilder/ChildrenSection/ChildrenSection.jsx

```diff
@@ -53,6 +54,8 @@ const styles = theme => ({
 export const ChildrenSectionCmp = ({section, classes, formik: {values, handleChange}}) => {
     const {t} = useTranslation();
     const {sections} = useContentEditorSectionContext();
+    const {nodeData} = useContentEditorContext();
+
     const canAutomaticOrder = values && values[Constants.automaticOrdering.mixin] !== undefined;
```

Comment:
I would move this test in FormBuilder.jsx
and pass the canAutomaticOrder as prop to the ChildrenSection.
Also we could create and other prop to have:

```
const canAutomaticOrder = values && values[Constants.automaticOrdering.mixin] !== undefined;
const canManualOrder = nodeData.primaryNodeType.hasOrderableChildNodes;
```

and pass this to the ChildrenSection.jsx as props, like:

`<ChildrenSection key={section.displayName} section={section} canAutomaticOrder={canAutomaticOrder} canManualOrder={canManualOrder}/>`